starwars/src/App.js

```diff
@@ -1,11 +1,14 @@
 import React, { Component } from 'react';
 import './App.css';
-
+//import Parent from './components/Parent';
 class App extends Component {
-  constructor() {
-    super();
+  constructor(props) {
```

Comment:
You don't actually need to pass props to the constructor and super. You only need to do that if you're accessing props inside the constructor function.